src/LoggerManager.F90

```diff
@@ -481,8 +482,9 @@ subroutine basic_config(this, unusable, filename, level, stream, force, handlers
       if (present(handlers)) then
          iter = handlers%begin()
          do while (iter /= handlers%end())
-            h => iter%get()
+            allocate(h, source = iter%get())
```

Comment:
This is not necessary.  The original code should be fine here.

Reply:
The handlers will disappear. We should copy the value out of it.

Reply:
Right.  I missed that.  But ... this instead becomes a memory leak.      Not sure if we should still add them to the config elements so that they will persist or ...

Reply:
Python "cheats" with garbage collection.

Reply:
There won't be memory leak if we deallocate those pointers at the end of clean up